pkg/csi/service/fcd/constants.go

```diff
@@ -38,4 +38,9 @@ const (
 	AttributeFirstClassDiskVcenter         = "vcenter"
 	AttributeFirstClassDiskDatacenter      = "datacenter"
 	AttributeFirstClassDiskPage83Data      = "page83data"
+	AttributeFirstClassDiskZone            = "zone"
+	AttributeFirstClassDiskRegion          = "region"
+
+	//Additional ENVVARs
+	GLOG_LEVEL = "GLOG_LEVEL"
```

Comment:
Now that this is set automatically, we don't need this anymore.

Reply:
Yep will remove it